Import in Signup: Auto-continue to the destination at end of flow

jblz opened the pull request:
#### Changes proposed in this Pull Request

* Auto-continue to the destination at end of flow

#### Testing instructions

* Run master
* Browse to `/start/import`
* Complete the flow
* Notice you have to click `Continue` at the end of the flow
* Run this branch
* Repeat the above
* Notice you do not need to click `Continue` at the end of the flow -- you are taken to the `destination`

jancavan commented:
Works great 👍

jancavan commented:
I don't think we need the disabled "Please wait..." button on here anymore. The message "Give us one minute..." is sufficient.

For those without accounts, we should just remove the verification notice:

![screenshot 2018-10-11 12 08 57](https://user-images.githubusercontent.com/4924246/46827890-a5f15580-cd4e-11e8-8663-e12cde2dfa06.png)

They will see the same notice once they land on `/import`:

![screenshot 2018-10-11 12 11 34](https://user-images.githubusercontent.com/4924246/46827964-d46f3080-cd4e-11e8-9fa0-84955a00fbe9.png)

jblz commented:
> I don't think we need the disabled "Please wait..." button on here anymore. The message "Give us one minute..." is sufficient.

addressed in #27805

> For those without accounts, we should just remove the verification notice

addressed in #27807
